presto-product-tests-launcher-idea-plugin/README.md

```diff
@@ -0,0 +1,34 @@
+# IntelliJ IDEA plugin for launching Presto product tests
+
+
+## Installing the plugin
+
+- In IntelliJ terminal (`View > Tool windows > Terminal`) run `./gradlew installToIdea`, or (in any terminal)
+use `./gradlew installToIdea -DideaUserHome=<path-to-idea-user-directory>`
+
+- Restart IntelliJ
+
+- In IntelliJ, in `Edit configurations > Templates > TestNG > Environment variables` set (for example)
+    `PT_ENVIRONMENT=singlenode;TESTCONTAINERS_REUSE_ENABLE=true`
+    
+- Mac only: add `/Applications/IntelliJ IDEA.app/Contents` to mountable directories in Docker settings
+
+- Run tests normally (for example by clicking green arrow in the editor); you can also use IntelliJ debugger
+
+
+## Building
+
+`./gradlew assemble`
```

Comment:
How does this fit into the build lifecycle of the whole project?

BTW, we don't use Gradle in Presto, so please migrate this to Maven.

Reply:
Gradle (or Scala sbt) are the only build tools supported by Intellij plugin sdk. 

It's integrated into build in ci.yml, for now you have to compile it manually. when it stabilizes, I guess it will be uploaded somewhere (S3? Maven repo?), so you could simply download it and put into plugins directory.